packages/bpk-component-card-list/src/BpkCardListRowRail/BpkCardListCarousel.module.scss

```diff
@@ -52,6 +58,31 @@
       overflow: visible;
       box-sizing: border-box;
       scroll-snap-align: start;
+
+      @include breakpoints.bpk-breakpoint-mobile {
+        &:first-child {
+          padding-left: tokens.bpk-spacing-sm();
+
+          @include utils.bpk-rtl {
+            padding-right: tokens.bpk-spacing-sm();
+            padding-left: tokens.bpk-spacing-md();
+          }
+        }
+      }
+
+      @include breakpoints.bpk-breakpoint-mobile {
+        flex: 0 0
+          calc(
+            (
+                100% -
+                  (
+                    tokens.bpk-spacing-md() *
+                      (var(--initially-shown-cards, 3) - 1)
+                  )
+              ) /
+              (var(--initially-shown-cards, 3) - 0.5)
```

Comment:
0.5 means that when on mobile, half of the last card in the visible area will be hidden.